website/sidebars.js

```diff
@@ -30,13 +30,18 @@ export default {
       items: [
         {
           type: "doc",
-          label: "STARK by hand",
-          id: "proof-system/stark-by-hand",
+          label: "Sequence Diagram",
+          id: "proof-system/proof-system-sequence-diagram",
         },
         {
           type: "doc",
-          label: "Sequence Diagram",
-          id: "proof-system/proof-system-sequence-diagram",
+          label: "Execution Traces",
+          id: "proof-system/what-is-a-trace",
+        },
+        {
+          type: "doc",
+          label: "STARK by Hand",
+          id: "proof-system/stark-by-hand",
```

Comment:
@pdg744 - not sure what order you want these in, but without them in the side bar we have errors, and those pages do not have a sidebar themselves

Reply:
Not super opinionated about the ordering here. 

I guess my instinct is that the "What is an Execution Trace" doc may fit better in the `docs/reference-docs/` section along side "about-arithmetic-circuits" etc, but happy to go ahead with whatever feels simplest at the moment in order to get the link-checker working, and worry about perfecting the organization later.